src/helpers/_font-faces.scss

```diff
@@ -21,47 +21,26 @@
 
       @font-face {
         font-family: "nta";
```

Comment:
I wonder if there's a chance that since the font-family is named the same thing that a service could accidentally load the old font into GOV.UK Frontend?

Might be safer to update the font-family name... something to test.